pages/library/add.js

```diff
@@ -0,0 +1,142 @@
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import { useRef } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { v4 as uuidv4 } from "uuid";
+
+const StyledForm = styled.form`
+  display: grid;
+  gap: 0.2rem;
+  flex-direction: column;
+  grid-template-columns: 50% 50%;
+  margin-top: 16rem;
+  color: black;
+  justify-content: center;
+  align-items: center;
+  font-family: Georgia, "Times New Roman", Times, serif;
+`;
+
+const StyledLabel = styled.label`
+  text-align: right;
+`;
+
+const StyledInput = styled.input`
+  background-color: transparent;
+`;
+const StyledButton = styled.button`
+  display: flex;
+  position: absolute;
+  margin-top: 2rem;
+  top: 23rem;
+  text-align: center;
+  justify-self: center;
+  justify-self: center;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-weight: bold;
+`;
+const StyledSelect = styled.select`
+  background-color: transparent;
+`;
+const StyledPopup = styled.div`
+  position: absolute;
+  border: 2px solid white;
+  top: 30rem;
+  left: 0;
+  text-align: center;
+  width: 100%;
+`;
+
+const StyledBook = styled(Image)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  object-fit: contain;
+  width: 100%;
+  height: 100%;
+  z-index: -2;
+`;
+const StyledBackToBookImage = styled(Image)`
+  position: relative;
+  margin-top: 10rem;
+`;
+const BackgroundImage = styled(Image)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  opacity: 0.7;
+  z-index: -3;
+  object-fit: cover;
+`;
+
+export default function AddWisdom({ handleNewWisdomSubmit }) {
+  const inputRef = useRef(null);
+  const [popupActive, setPopupActive] = useState(false);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const wisdomData = Object.fromEntries(formData);
+    handleNewWisdomSubmit({ ...wisdomData, answeredRight: 0, id: uuidv4() });
+    setPopupActive(true);
+    setTimeout(() => {
```

Comment:
Very small point. But we could change this to be. 

`setTimeout(() => setPopupActive(false), 1500);`